src/sdk.ts

```diff
@@ -108,45 +108,6 @@ export declare type Feature = {
   };
 };
 
-export enum TransportSubmode {
-  AIRPORT_LINK_BUS = "airportLinkBus",
-  AIRPORT_LINK_RAIL = "airportLinkRail",
-  CITY_TRAM = "cityTram",
-  DOMESTIC_FLIGHT = "domesticFlight",
-  EXPRESS_BUS = "expressBus",
-  FUNICULAR = "funicular",
-  HELICOPTER_SERVICE = "helicopterService",
-  HIGH_SPEED_PASSENGER_SERVICE = "highSpeedPassengerService",
-  HIGH_SPEED_VEHICLE_SERVICE = "highSpeedVehicleService",
-  INTERNATIONAL = "international",
-  INTERNATIONAL_CAR_FERRY = "internationalCarFerry",
-  INTERNATIONAL_COACH = "internationalCoach",
-  INTERNATIONAL_FLIGHT = "internationalFlight",
-  INTERNATIONAL_PASSENGER_FERRY = "internationalPassengerFerry",
-  INTERREGIONAL_RAIL = "interregionalRail",
-  LOCAL = "local",
-  LOCAL_BUS = "localBus",
-  LOCAL_CAR_FERRY = "localCarFerry",
-  LOCAL_PASSENGER_FERRY = "localPassengerFerry",
-  LOCAL_TRAM = "localTram",
-  LONG_DISTANCE = "longDistance",
-  METRO = "metro",
-  NATIONAL_CAR_FERRY = "nationalCarFerry",
-  NATIONAL_COACH = "nationalCoach",
-  NIGHT_BUS = "nightBus",
-  NIGHT_RAIL = "nightRail",
-  RAIL_REPLACEMENT_BUS = "railReplacementBus",
-  REGIONAL_BUS = "regionalBus",
-  REGIONAL_CAR_FERRY = "regionalCarFerry",
-  REGIONAL_RAIL = "regionalRail",
-  SCHOOL_BUS = "schoolBus",
-  SHUTTLE_BUS = "shuttleBus",
-  SIGHTSEEING_BUS = "sightseeingBus",
-  SIGHTSEEING_SERVICE = "sightseeingService",
-  TELECABIN = "telecabin",
-  TOURIST_RAILWAY = "touristRailway"
-}
-
 export declare enum TransportSubmode {
```

Comment:
Huh, you made me look up the documentation for [`declare`](https://github.com/Microsoft/TypeScript/blob/v1.8.10/doc/spec.md#11-ambient-declarations) here, but I guess this is a remnant of a .d.ts file, that probably should be removed. Same with `Feature` on [line 82](https://github.com/AtB-AS/mittatb-app/blob/ridwan/fix-rail-replacement-bus-text-missing/src/sdk.ts#L82). 

```suggestion
export enum TransportSubmode {
```

Reply:
Updated both enums, thank you!